cirq-core/cirq/work/collector.py

```diff
@@ -163,53 +165,56 @@ async def collect_async(
         Returns:
             The collector's result after all desired samples have been
             collected.
-
-        See Also:
-            Python 3 documentation "Coroutines and Tasks"
-            https://docs.python.org/3/library/asyncio-task.html
         """
-        pool = work_pool.CompletionOrderedAsyncWorkPool()
+        results = duet.AsyncCollector[Tuple[CircuitSampleJob, 'cirq.Result']]()
```

Comment:
what is going on here? Is this a form of type annotation I've never seen before or does this actually change the object that has been created?

Reply:
ok, glad it's just a type hint. my slight preference would be for the type annotation at the cost of some repetition but I defer to you

Reply:
Switched to type annotation. It still _just_ fits on one line :-)